frictionless/eia860.py

```diff
@@ -11,26 +11,10 @@
     "description":
         "US Energy Information Administration (EIA) Form 860 data for electric power "
         "plants with 1 megawatt or greater combined nameplate capacity.",
-    "profile": "data-package",
-    "keywords": [
-        "electricity", "electric", "boiler", "generator", "plant", "utility",
-        "fuel", "coal", "natural gas", " prime mover", "eia860", "retirement",
-        "capacity", "planned", "proposed", "energy", "hydro", "solar", "wind",
-        "nuclear", "form 860", "eia", "annual", "gas", "ownership", "steam",
-        "turbine", "combustion", "combined cycle", "eia",
-        "energy information administration"
-    ],
     "licenses": [licenses.us_govt, ],
```

Comment:
`RESOURCE_METADATA` does not specify licenses for any of the resources so “cc-by-4.0” is used by default. Should we add licenses to each PUDL resource?

Reply:
IIRC it uses CC-BY-4.0 by default, but that can be overridden with a specific license. If you think it would be clearer to specify it within the resource definitions we could do that.  I think the most important thing is that any data we publish have a clear license, and that it be defensible (i.e. we don't accidentally apply an incorrect license to any data we're publishing)

Reply:
I've explicitly associated a `license_raw` and `license_pudl` with each `DataSource` now, and they are required fields, since we should never be publishing any data that doesn't have clear licensing.